Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ghcr.io/containerbase/base:9.8.2@sha256:f41dc51323ab4839fa7e1242750363497fa45aec9d08f93ce2b42c5b10a581e7 AS base
```

Comment:
needs to be multi stage to build full image too. should depend on build arg.

Reply:
Can we defer this change to a following PR?

Reply:
ok, but disable push to registry until we've proper image configurations.